components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -552,18 +553,19 @@ define([
          */
         updateEvaluationLabel: function() {
             this._evalLabel && this._evalLabel.remove();
+            var gender = this.getNode().getGender();
             if (this.getNode().getEvaluated()) {
                 if (this.getNode().getLifeStatus() == 'aborted' || this.getNode().getLifeStatus() == 'miscarriage') {
                     var x = this.getX() + this._shapeRadius * 1.6;
                     var y = this.getY() + this._shapeRadius * 0.6;
                 }
                 else {
                     var mult = 1.1;
-                    if (this.getNode().getGender() == 'U') mult = 1.3;
-                    else if (this.getNode().getGender() == 'M') mult = 1.4;
-                    if (this.getNode().isProband) mult *= 1.1;
-                    var x = this.getX() + this._shapeRadius*mult - 5;
-                    var y = this.getY() + this._shapeRadius*mult;
+                    if (gender == 'U' || gender == 'O') {mult = 1.3;}
+                    else if (gender == 'M') {mult = 1.4;}
+                    if (this.getNode().isProband) {mult *= 1.1;}
+                    var x = this.getX() + this._shapeRadius*mult;
+                    var y = this.getY() + this._shapeRadius*mult - 10;
```

Comment:
All of them or just the one I have changed? I am not sure I can list all justifications @allasm could have in mind to correctly explain his parameters.